pkg/printer/plaintext.go

```diff
@@ -0,0 +1,12 @@
+package printer
+
+import (
+	"fmt"
+	"io"
+)
+
+// PrintPlaintext is a printer that prints the provided value as is
+func PrintPlaintext(out io.Writer, v interface{}) error {
+	fmt.Fprintf(out, `%#v`, v)
```

Comment:
I just tried it out on the [go playground](https://play.golang.org/p/zoN94WdrPS2) and realized that my suggestion of `%#v` won't work well with what we are trying to do with helm status. It's going to put quotes around the passed value when the goal is the pass through the helm output as-is.

Sorry about that, let's go back to what you had before. I don't recall if it was `fmt.Fprintln(out, v)` which adds a trailing newline or `fmt.Fprintf(out, "%v", v)` which relies on the passed input having a newline at the end.

Reply:
Ah good point; switched back to `fmt.Fprintf(out, "%v", v)`